examples/apps/contact-collection/package.json

```diff
@@ -12,12 +12,11 @@
   "license": "MIT",
   "author": "Microsoft and contributors",
   "main": "dist/index.js",
-  "module": "lib/index.js",
+  "module": "dist/index.js",
```

Comment:
Should we be consistent and keep the "lib" output for `esnext` module target? (and remove `dist` directory instead)

Reply:
done.